community/CM-Configuration-Management/acm-app-pv-backup/resources/policies/oadp-hdr-app-backup.yaml

```
@@ -37,11 +37,11 @@ spec:
                 apiVersion: velero.io/v1
                 kind: Schedule
                 metadata:
-                  name: '{{hub $configMap.data.backupPrefix hub}}-{{hub $configMap.data.backupVolumeSnapshotLocation hub}}-{{ fromClusterClaim "name" }}'
+                  name: '{{hub $configMap.data.backupPrefix hub}}-{{hub $configMap.data.backupVolumeSnapshotLocation hub}}-{{hub (printf "%s" .ManagedClusterName) hub}}'
```

nit: `.ManagedClusterName` already returns a string, so `printf` isn't necessary.

```suggestion
                  name: '{{hub $configMap.data.backupPrefix hub}}-{{hub $configMap.data.backupVolumeSnapshotLocation hub}}-{{hub .ManagedClusterName hub}}'
```

---

I suppose an alternative would be to use `printf` for all of it in a single template, but the result is the same and I'm not sure there's any advantage to either route:

```suggestion
                  name: '{{hub printf "%s-%s-%s" $configMap.data.backupPrefix  $configMap.data.backupVolumeSnapshotLocation  .ManagedClusterName) hub}}'
```